src/sentry/conf/server.py

```diff
@@ -1369,6 +1369,14 @@ def create_partitioned_queues(name):
 )
 
 # Delay (in ms) to induce on API responses
+#
+# Simulates a small amount of lag which helps uncover more obvious race
+# conditions in UI interactions. It's also needed to test (or implement) any
```

Comment:
Shouldn't we add a random +/- difference to the base value if we want to find race conditions with this?

Reply:
It's not that kind of race - its more like, the UI renders and doesnt account for the fact that an API request might not return instantaneously. So you end up in a weirdly broken UI state that you don't notice in most situations. Variability already exists on the server, its just more about forcing _some_ kind of delay that is human noticeable.